DirectEmployers/MyJobs
# [PD 3048 view all option persist]

## What's this PR do?
PR persist the pagination value of All Pages when data is changed inside of the app

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics app

## How should this be manually tested?
Pull down the repository. Go inside of the analytics app and begin filtering to where there is pagination. Select All for the Page Size and then go into the Calendar and tinker with the dates and move them around. You should get different ALL values for the amount of rows in the table and the data should persist inside

## Any background context you want to provide?
https://directemployersfoundation.atlassian.net/browse/PD-3048
